languoids/tree/book1242/ndon1252/md.ini

```diff
@@ -44,7 +44,7 @@ multitree = njd
 [hh_ethnologue_comment]
 comment = E16/E17/E18/E19/E20/E21/E22/E23/E24 has an entry Ndonde Hamba [njd] in the Lindi region of Tanzania.  While different from its immediate neighbours,
 	it is a dialect of the language in the Makonde [kde] entry
-	(**hh:g:Kraal:Makonde**, **hh:hlde:Crosse-Upcott:KiNgindo**:13-14, 351-358) See also: Makonde [kde].
+	(**hh:g:Kraal:Makonde**, **hh:hlde:Cross-Upcott:KiNgindo**:13-14, 351-358) See also: Makonde [kde].
```

Comment:
@d97hah I fixed this already last time, but it keeps coming back. It's *Cross**e***.